ansys/mapdl/core/mapdl.py

```diff
@@ -2230,6 +2230,8 @@ def run(self, command, write_to_log=True, mute=None, **kwargs):
         text = self._run(command, mute=mute, **kwargs)
 
         if mute:
+            self._response = None
+            self._log.info("The response is being muted.")
```

Comment:
Out of scope for this PR. Also, this may lead to a ton of log messages. If, in the future, we do add this in, we should add that the "response of <command> is being muted", but I think it's not necessary.
```suggestion
```